woocommerce/woocommerce-android
# Enable card present payments on all stores

Closes: #4833


### Description
Removes dynamic (api driven) feature flag for in person payments => enables in person payments menu item and collect payment button on all stores.

### Testing instructions

1. Switch to a store which isn't enrolled in the beta
2. Make sure "In Person Payments" menu item is visible in settings
3. Make sure "Collect Payment" button is visible on eligible orders
